WordPress/src/main/java/org/wordpress/android/ui/media/MediaBrowserActivity.java

```diff
@@ -544,8 +544,13 @@ public void deleteMedia(final ArrayList<Integer> ids) {
     }
 
     private void uploadList(List<Uri> uriList) {
+        if (uriList == null || uriList.size() == 0) {
```

Comment:
nit: I think the `uriList.size() == 0`  check here is not important as it will be checked by the `for` statement below.